fastai/data/external.py

```diff
@@ -110,15 +112,24 @@ class URLs():
     WT103_FWD          = f'{S3_MODEL}wt103-fwd.tgz'
     WT103_BWD          = f'{S3_MODEL}wt103-bwd.tgz'
 
-    def path(url='.', c_key='archive'):
+    def path(
+        url='.', # URL of file to download
+        c_key='archive' # Name of a folder in `fastai_cfg` to save the file. Should be one of [`'data', 'archive', 'storage', 'model'`]
+    ) -> Path: # Path containing where the file will be downloaded to
         "Return local path where to download based on `c_key`"
```

Comment:
We might want to improve docstrings too as we go. Docstrings shouldn't generally say "return", since that's assumed.

Reply:
Done